safe_transaction_service/history/migrations/0058_alter_webhook_url.py

```diff
@@ -16,9 +16,7 @@ class Migration(migrations.Migration):
             name="url",
             field=models.CharField(
                 max_length=255,
-                validators=[
-                    safe_transaction_service.history.models._validate_webhook_url
-                ],
+                validators=[safe_transaction_service.history.utils.validate_url],
```

Comment:
@safe-global/core-api not sure if better approach would be remove this validator because the `_validate_webhook_url` was used just here.

Reply:
Sorry, I'm not understanding this

Reply:
I mean, the function looks used just in the migration and maybe each time that a webhook is inserted, as we are removing the model this does not make sense, I just leave it because if not we will break the django migrations history. 
But how we are squashing the migrations in other PR this question is not relevant :)

Reply:
Oh ok, makes sense!